### PR TITLE
Angel Help App

### PR DESCRIPTION
**Pull request by Baroka-wp**
Build an app allows the user (passionate about spirituality) to filter angel's list by the concerns he wants to solve. It can be Health, Money, Love, or Job concern. 
- [x] fetch 72-angels-of-kabala [r](https://the-72-angels-of-the-kabbalah.onrender.com/)
- [x] Create the structure of react file
- [x] Build reducer and connect it with React
- [x] Display the category of Angels in the root page
- [x] Make Filter action by category
- [x] Display Angel Details
- [x] Make tests for components and reducer 
- [x] Write documentation of App (on READEME.md)
- [x] Make presentation of the App with loom
- [x] Deploy App on GitHub Page

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,207 @@
...
+  position: absolute;
+  height: 20vh;
+  background: url(./images/angel.jpeg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.card.ligth-card {
+  background: #cd4275 !important;
```
Comment (prolajumokeoni) [inline src/App.css]: - [ ] Please do not overuse !important rule in your CSS style definitions, as this is not a good practice when writing code 👍🏿

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,207 @@
...
+}
+
+.card.ligth-card:hover {
+  background: linear-gradient(90deg, rgba(0, 0, 0, 0.082) 8%, rgba(236, 78, 138, 0.4285714285714286) 100%) !important;
```
Comment (prolajumokeoni) [inline src/App.css]: - [ ] Please do not overuse !important rule in your CSS style definitions, as this is not a good practice when writing code 👍🏿

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,207 @@
...
+.angelsList {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 30px;
+  background: #ec4e8a;
+}
+
+.angelsList.ligth-card {
+  background: #cd4275 !important;
```
Comment (prolajumokeoni) [inline src/App.css]: - [ ] Please do not overuse !important rule in your CSS style definitions, as this is not a good practice when writing code 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Baroka-wp, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️

- [ ] Kindly use the Typography: font face, size and weight specified in the [project requirements](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) in your design 👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Baroka-wp** [comment]
Thank you very much @prolajumokeoni for your review. According to you review I 
- [x] Add Lato font as requiered
- [x] Not used `important!` in css

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Baroka-wp, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Baroka-wp** [comment]
## Thank you @prolajumokeoni

**Pull request merged**